src/components/ec-text-field/ec-text-field.spec.js

```diff
@@ -0,0 +1,43 @@
+import { mount } from '@vue/test-utils';
+import EcTextField from './ec-text-field.vue';
+import { withMockedConsole } from '../../../tests/utils/console';
+
+describe('EcTextField', () => {
+  function mountTextField(props, mountOpts) {
+    return mount(EcTextField, {
+      propsData: {
+        value: 'Text test',
+        type: 'text',
+        errorMessage: 'error msg',
+        state: '',
+        label: 'label test',
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should throw if no props were given', () => {
```

Comment:
The test is testing something else. You are actually passing all props. It's validations that are failing. 

In this test, call `mount(EcTextField)` and check the error, then create another test where you  `mountTextField({ type: 'random', state: 'random' });` and call it "should validate given props".